### DISCUSSION TITLE
Trying to use UDM-IPTV but i'm not getting it to work

### DISCUSSION THREAD

**Discussion by marcoheijkoop** [fabianishere/udm-iptv#246] [Q&A]
Hi Everyone,

I'm trying to get UDM-IPTV working but i'm not getting it to work.
At this moment my config is like this: https://dawnbringer.net/blog/1086/ubiquiti-usg-kpn-experia-iptv
And this is working great. Today i had a malfunction with KPN so i tried to get rid of my Experiabox v10a.

I changed the WAN settings and when KPN was back up i had internet.
I was able to execute the commando: 
![image](https://user-images.githubusercontent.com/11247178/227785254-88ff9158-cc8a-4be5-869c-bf5e0c2a3018.png)
I went thru the steps and use 192.168.1.1 as the netwerk for IPTV. Before it was 192.168.2.1 coming from the Experiabox v10a
I tried to reboot the IPTV receivers but there was no TV. So i did a roll-back and everything is working again
I think i'm  missing something but i don't know what is it is. After the install off the script i didn't reboot the UDM
Do i have to do that?

From the Experiabox i have an VLAN4+6 Uplink and all the IPTV Receivers are in that VLAN. 
The only thing is that DHCP for the receivers comes from the Experiabox with IP 192.168.2.1. And it's not used with in UDM.

I'm missing something but i don't know what is.
Must i create a new netwerk with IP Range 192.168.2.1? Or can i use 192.168.1.1 for this with out problems?

Hope someone with more knowledge then me can help me.

Best Regards
Marco

**Comment by fabianishere** [comment]
Just to be sure, you are using an UDM (Pro) and not a USG (Pro)?

The default KPN config only works with the Experiabox removed and you have to make sure to enable the VLAN on which your IPTV receivers live. If `192.168.1.0/24` is the subnet of your native LAN, you need to make sure to enable `br0` in the config.

Please also provide the output of `udm-iptv diagnose` so we can better assist you.

**Reply by thomt44** [reply]
Fabian must the iptv be on the default ip?
I want it on a different lan. I created it but the tv only works on the default

**Reply by thomt44** [reply]
And how do a reinstall as the first time? Im missing some steps after uninstall and reinstall. Its not the same process

**Reply by thomt44** [reply]
Solved!. With spacebar select the right vlan in the reconfigure commnd

**Comment by marcoheijkoop** [comment]
Good question, I do indeed have a USG Pro here somewhere. ;-) But yes i'm using a UDM Pro.
Ok. I'm going to enable  br0  in the config next time. I have 192.168.1.0/24 as native lan.
The rest of the network has more VLAN's

I'm going to plan this because the family is very critical about watching TV. So i must do it when they are not at home.
If  i'm still having problems then i will run udm-iptv diagnose

**Reply by marcoheijkoop** [reply]
Hi Fabian,

Got it working.

I'm only thought that i saw something about making an backup of the settings to a persistent location. So after a upgrade off the UDM-Pro you can put the settings back. But i can't find it anymore.

Is this still the case?

**Reply by fabianishere** [reply]
The advice is still there, but the command has been removed as it was causing too many people trouble.
